src/main/java/com/techsophy/tsf/runtime/form/dto/ComparatorOperation.java

```diff
@@ -0,0 +1,31 @@
+package com.techsophy.tsf.runtime.form.dto;
+
+import lombok.Data;
+import org.springframework.data.mongodb.core.query.Criteria;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Data
+public class ComparatorOperation implements FilterOperation
+{
+    private Integer lt;
```

Comment:
this can be of type number

Reply:
yes changing the datatype from Integer to Number